analysis/priestdiscipline/src/modules/shadowlands/legendaries/ClarityOfMindEvang.tsx

```diff
@@ -0,0 +1,140 @@
+import SPELLS from 'common/SPELLS';
+import { SpellLink } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import ItemManaGained from 'parser/ui/ItemManaGained';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+import { POWER_WORD_SHIELD_ATONEMENT_DUR } from '@wowanalyzer/priest-discipline/src/constants';
+
+import AtonementAnalyzer, { AtonementAnalyzerEvent } from '../../core/AtonementAnalyzer';
+
+const CLARITY_EXTENSION_DURATION = 6000;
+const EVANG_EXTENSION_DURATION = 6000;
+const POWER_WORD_SHIELD_MANA_COST = 1550;
+
+class ClarityOfMindEvang extends Analyzer {
+  private atonementHealing: number = 0;
+  raptureBuffActive = false;
+  raptureShields = [] as any;
```

Comment:
this should have a type declaration instead of any.

for example, add this outside the class:

```ts
type RaptureShieldInfo = {
  applyBuff: ApplyBuffEvent;
  extendedByEvang?: boolean;
}
```

then this becomes: `raptureShields: RaptureShieldInfo[] = [];`

then the uses of `any` throughout the file can be removed.